src/app/static/src/app/store/adrUpload/actions.ts

```diff
@@ -91,7 +90,7 @@ export const actions: ActionTree<ADRUploadState, RootState> & ADRUploadActions =
                 break
             }
         }
-        await dispatch("adr/getAndSetDatasets", selectedDatasetId);
-        dispatch("getUploadFiles");
+        await dispatch("adr/getAndSetDatasets", selectedDatasetId, {root: true});
+        await dispatch("getUploadFiles");
```

Comment:
Doesn't really matter, but why add this `await`?

Reply:
Simply to silence IDE warnings. But even if it's fire-and-forget in this case (is there an alternative idiom?) I guess it's a good habit for cases when it does matter...